.pre-commit-config.yaml

```diff
@@ -3,16 +3,18 @@ default_language_version:
   node: 15.1.0
 repos:
   - repo: https://github.com/python/black
-    rev: 23.7.0
+    rev: 24.10.0
```

Comment:
No - we should stick with one version for all of these for a sustained period of time and only do occasional updates.

Linters and style-enforcers are moving targets and introduce a lot of changes.

For this PR please revert these versions - and then ensure that the code follows the conventions of those version, so that we don't introduce a lot of diffs with the other ongoing work.

Reply:
Make sense, reverted